lib/manageiq/session.rb

```diff
@@ -0,0 +1,48 @@
+require "extensions/session_extension"
+
+module ManageIQ
+  module Session
+    FakeRequest = Struct.new(:env)
+
+    def self.store
+      @store ||= fetch_store_from_rails
+    end
+
+    def self.store=(session_store)
+      adapter_klass = "ManageIQ::Session::#{session_store.to_s.camelize}Adapter".safe_constantize
+      raise ArgumentError, "invalid session store adapter: #{session_store.inspect}" unless adapter_klass
+
+      configure_session_store(adapter_klass.new)
+
+      @store = nil
+    end
+
+    def self.configure_session_store(adapter)
+      Vmdb::Application.config.session_store(*adapter.session_store_config_args)
+      msg = "Using session_store: #{Vmdb::Application.config.session_store}"
+      _log.info("MIQ(SessionStore) #{msg}")
```

Comment:
```suggestion
      _log.info(msg)
```

_log gives us the prefix automatically